### ISSUE TITLE
Withdrawals make auto-move unusable

### ISSUE THREAD

**Issue by ErgosphericMeasure** [CleverRaven/Cataclysm-DDA#75488]
### Describe the bug

When using auto-move while having withdrawals (from drugs, alcohol, etc.) the "You start having withdrawals! Cancel auto move?" popup appears after every step, rendering auto-move near-entirely pointless.

### Attach save file

[New Britain-trimmed.tar.gz](https://github.com/user-attachments/files/16514181/New.Britain-trimmed.tar.gz)

### Steps to reproduce

1. Make a character
2. Take a bunch of meth
3. Wait for withdrawals
4. Try to auto-travel anywhere

### Expected behavior

Withdrawal warning should probably appear only once during any given travel session.

### Screenshots

![Screenshot from 2024-08-06 14-06-12](https://github.com/user-attachments/assets/f380d3ee-eb03-48b9-bf7e-27ec76ea4b89)

### Versions and configuration

- OS: Linux
    - OS Version: LSB Version: n/a; Distributor ID: EndeavourOS; Description: EndeavourOS Linux; Release: rolling; Codename: rolling; 
- Game Version: cdda-experimental-2024-08-05-0511 6ee8e16 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth]
]

**Comment by Consoleable** [comment]
Theres no easy way to fix this until addiction is overhauled or automove is fixed. Afaict breaking automove is actually a problem with automove itself, and how it handles distractions, as safe mode distractions can break it as well. 
Withdrawals are triggered on a 6 hour timer(which might be longer or shorter based on traits), try waiting after the withdrawal notification is triggered for a minute and then continuing automove. Alternatively, satisfying your addiction with some more drugs should help. If you just don't care about your characters withdrawals then yeah, turning off the distraction makes sense. 
The withdrawal distraction is mainly for preventing people from getting locked into trying to craft something until they pass-out.

**Comment by ErgosphericMeasure** [comment]
Yeah, I just disabled the distraction myself to deal with it, but figured I should document the interaction, as there's no way the popup firing every step is the best way for those features to interact.
